src/Equinox.MessageDb/MessageDbClient.fs

```diff
@@ -11,10 +10,11 @@ open NpgsqlTypes
 
 exception WrongExpectedVersion
 
-type MessageDbClient(source: NpgsqlDataSource) =
+type MessageDbClient(source: unit -> Task<NpgsqlConnection>) =
     member __.ReadStream(streamName: string, fromPosition: int64, batchSize: int64, ct) =
```

Comment:
we assume a V6 compiler so `__` -> `_` (Rider highlights this)
Also for all fresh code, `streamName: string` becomes `streamName : string`  
(Open to editorconfig driven forcing of this - fantomas is probably a step too far atm)